src/components/Graph/Graph.tsx

```diff
@@ -1,27 +1,18 @@
 import React, { FC, useCallback, useEffect, useMemo, useRef, useState } from 'react';
-
 import graphLayout from 'visualization/layout';
-import Node from 'components/Node';
-import Arrow from 'components/Arrow';
 import { evaluatePatientOnPathway } from 'engine';
-import { EvaluatedPathway, PathwayResults, DocumentationResource } from 'pathways-model';
-import { Layout, NodeDimensions, Edge } from 'graph-model';
+import { EvaluatedPathway, PathwayResults } from 'pathways-model';
+import { Layout, NodeDimensions } from 'graph-model';
 import { usePatientRecords } from 'components/PatientRecordsProvider';
 import { DomainResource } from 'fhir-objects';
-
+import GraphRender from 'components/Graph/GraphRender';
```

Comment:
use a relative import here because it's in the same directory.  GraphRender shouldn't be needed outside of Graph anyways since it's just an implementation detail.

```suggestion
import GraphRender from './GraphRender';
```